src/lib/integration_test_local_engine/swarm_network.ml

```diff
@@ -0,0 +1,561 @@
+open Core
+open Async
+open Integration_test_lib
+
+(* Parts of this implementation is still untouched from the Kubernetes implementation. This is done
```

Comment:
We should work on extracting the graphql definitions from the engine layer. Those should be in the shared `Integration_test_lib` library, and the engine should just be in charge of the transport logic for sending the queries. We can revisit this later though, not in this PR.